mypy/main.py

```diff
@@ -222,6 +222,11 @@ def process_options(args: List[str],
     # which will make the cache writing process output pretty-printed JSON (which
     # is easier to debug).
     parser.add_argument('--debug-cache', action='store_true', help=argparse.SUPPRESS)
+    # --half-assed mode is a bit like --silent-imports in that it distinguishes
+    # modules listed on the command line; however instead of ignoring such
+    # imports, it follows them but silences errors.  Stubs are unaffected.
```

Comment:
Blockers still abort the run. If both flags are set, indeed we process the module silently if found, and skip it silently if not found.